src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java

```diff
@@ -79,6 +80,18 @@ public void setPanel(BasePanel panel) {
         this.metaData = panel.getBibDatabaseContext().getMetaData();
     }
 
+    public void updateEnableStatus() {
+        DatabaseLocation location = panel.getBibDatabaseContext().getLocation();
+        boolean isNotShared = (location != DatabaseLocation.SHARED);
+        encoding.setEnabled(isNotShared);
```

Comment:
I assume, the encoding is always UTF-8. Please put that as comment above this line.